CHANGELOG.md

```diff
@@ -1,8 +1,36 @@
 ## Next release
 
+## 0.39.0 / 2020-05-06
+
+* [CHANGE] Introduce release schedule (#3135)
```

Comment:
not sure we want this in the changelog?

Reply:
We had questions like "when will be the next release"? This should send a message that we have a cadence now.

Reply:
Oki. For better visibility it could even go as a preamble to the list of items.

Reply:
I think it's fine to include it.  I'd rather we err on the side of being too verbose.